src/site/content/en/fast/codelab-cloudinary-url-api/index.md

```diff
@@ -0,0 +1,334 @@
+  <div class="price">Violet bouquet- $9</div>
+</div>
+```
+
+This helps save more than 90% on image size.
+
+<div class="w-table-wrapper">
+  <table>
+    <tbody>
+      <thead>
+        <tr>
+          <th>
+            <p>
+              <a
+                href="https://codelab-cloudinary.glitch.me/images/flower1.png"
+                target="_blank"
+                rel="noopener noreferrer"
+                ><img src="https://codelab-cloudinary.glitch.me/images/flower1.png" width="100%"
+              /></a>
+            </p>
+            <strong>289 KB</strong> (Original)
+          </th>
+          <th>
+            <p>
+              <a
+                href="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://codelab-cloudinary.glitch.me/images/flower1.png"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <img
+                  src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto,a_ignore/https://codelab-cloudinary.glitch.me/images/flower1.png"
+                  width="100%"
+                />
+              </a>
+            </p>
+            <strong>22 KB</strong> (Cloudinary)
+          </th>
+        </tr>
+      </thead>
+    </tbody>
+  </table>
+</div>
+
+The photo on the right is 92.39% smaller than the one on the left, yet would probably look identical to the average user.
+
+{% Aside 'objective' %}
+  Now prepend all image links in the `index.html` with `https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/`. Make sure to change `demo` to your `cloud_name`.
+{% endAside %}
+
+## ✔︎ Check-in
+
+Your `index.html` file should now look like this:
+
+```html
+<!--
+Copyright 2018 Google LLC.
+SPDX-License-Identifier: Apache-2.0
+-->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Cloudinary URL API Demo: Optimized Images</title>
+    <meta name="Description" content="In this codelab, learn how to use Cloudinary with its simple URL API to optimize images for faster downloads and serve media in next-gen formats. All of that happens dynamically in the Cloud without any build step.">
+    <link href="https://cloudinary-res.cloudinary.com/image/asset/favicon-192x192-d6a96e11dd5adfebbb717d154665ee80.png" rel="icon" sizes="192x192" type="image/png">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>body {text-align: center}img {border: 1px solid black}.wrapper {margin-bottom: 24px}.wrapper .price {font-size: 24px}</style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Foobar Flowershop
+      </h1>
+      <p>We sell the best flowers!</p>
+    </header>
+    <div class="wrapper">
+      <img src="https://res.cloudinary.com/demo/image/fetch/q_auto,f_auto/https://cloudinary-url-api.glitch.me/images/flower1.png" alt="Yellow bouquet" />
```

Comment:
cloudinary-url-api -> codelab-cloudinary

Reply:
Ah dang it. Sorry about this. Last minute glitch name change, I thought I fixed all of these.

Reply:
🐛 FIX: 0988735